ERCS/erc-7943.md

```diff
@@ -0,0 +1,518 @@
+---
+eip: 7943
+title: uRWA - Universal Real World Asset Interface
+description: A minimal interface for regulated assets, targeting the broad spectrum of RWAs.
+author: Dario Lo Buglio (@xaler5)
+discussions-to: https://ethereum-magicians.org/t/erc-universal-rwa-interface/23972
+status: Draft
+type: Standards Track
+category: ERC
+created: 2025-06-02
+requires: 165
+---
+
+## Abstract
+
+This EIP proposes the Universal RWA (uRWA) standard, a minimal interface for tokens based on common standards like [ERC-20](./eip-20.md), [ERC-721](./eip-721.md) or [ERC-1155](./eip-1155.md), intended as a primitive for various classes of Real World Assets. It defines essential functions and events for regulatory compliance and enforcement actions common to RWAs. It also adopts [ERC-165](./eip-165.md) for introspection.
```

Comment:
This doesn't really explain what a Universal RWA standard actually is. What can you do with this proposal? You need to remember that not everyone coming to your standard will be intimately familiar with RWAs&mdash;I'm certainly not. :sweat_smile: